src/harness/client.ts

```diff
@@ -568,15 +568,17 @@ export class SessionClient implements LanguageService {
         return notImplemented();
     }
 
-    findRenameLocations(fileName: string, position: number, findInStrings: boolean, findInComments: boolean, providePrefixAndSuffixTextForRename?: boolean): RenameLocation[] {
+    findRenameLocations(fileName: string, position: number, findInStrings: boolean, findInComments: boolean, preferences: UserPreferences | boolean | undefined): RenameLocation[] {

```

Comment:
Not a big deal, but you might not have to handle `boolean` in these internal test-only implementations. I could be wrong though; maybe there’s a reason we need it?

Reply:
Because this client implements `LanguageService`, and so this method needs to be able to accommodate both overloads.

Reply:
👍 I couldn’t remember if that was a rule for implementing interfaces. Makes sense.